simplemodels/models.py

```diff
@@ -116,16 +116,11 @@ def __iter__(self):
     def __len__(self):
         return len(self._fields)
 
-    def as_dict(self):
-        result = dict()
-        for field_name, field in self.items():
-            if isinstance(field, Document):
-                field = field.as_dict()
-            elif isinstance(field, MutableSequence):
-                field = list(field)
-            result[field_name] = field
-
-        return result
+    def to_python(self):
```

Comment:
I thought that `as_dict` method remains but internally will iterate over fields and call `field.to_python()`. I clearly like `as_dict` as a serialization method.